airbyte-config/config-models/src/main/java/io/airbyte/config/EnvConfigs.java

```diff
@@ -946,6 +950,11 @@ public boolean shouldRunConnectionManagerWorkflows() {
     return getEnvOrDefault(SHOULD_RUN_CONNECTION_MANAGER_WORKFLOWS, true);
   }
 
+  @Override
+  public boolean shouldRunNotifyWorkflows() {
+    return getEnvOrDefault(SHOULD_RUN_NOTIFY_WORKFLOWS, true);
```

Comment:
I missed that but should this be false by default so we avoid releasing that to OSS before testing it?

Reply:
I had to think about this - I was originally thinking about one env variable for AUTO_DETECT_SCHEMA_CHANGES overall, but maybe we can use this separately as well to test just this workflow, so I think it's a good idea

Reply:
We should avoid adding to this class and instead use Micronaut's configuration property management via the `application.yml` file.